scenario/matchTag.yml

```diff
@@ -0,0 +1,32 @@
+input:
```

Comment:
This file should match the casing of the other files in this directory (snake_case.yml)